Formula/libident.rb

```diff
@@ -21,7 +21,14 @@ class Libident < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "f80a93f7750a66e987f21af7db62ba4f72c2c277036049915d3c8e6a8b044cf2"
   end
 
+  on_macos do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   def install
+    system "autoreconf", "-fvi" if OS.mac?
```

Comment:
```suggestion
    system "autoreconf", "--force", "--install", "--verbose" if OS.mac?
```
Let's use the long flags here (and the other PRs).